web/src/settings_org.ts

```diff
@@ -1142,6 +1157,8 @@ export function save_organization_settings(
                 // a change.
                 unsaved_welcome_message_custom_text = "";
             }
+            resize_settings_textarea("id_realm_welcome_message_custom_text");
```

Comment:
Instead of calling this for changing every setting, I think the function can take subsection element as input and then return early if there is no textarea in the subsection and resize them if there is one. 

And for the function calls in `settings_panel_menu`, there can be two ways - it can just call a separate function which can call this function in loop for all the subsections in section (this will handle other textarea elements added in future without having to update the code for them) or it can just call this function for two subsections based on the `section` value.